the-blue-alliance-ios.xcodeproj/project.pbxproj

```diff
@@ -714,13 +726,14 @@
 				145C57BB1ED3C178004955B2 /* MatchBreakdownViewController.swift in Sources */,
 				92942DBB1E215B8D008E79CA /* MyTBAViewController.swift in Sources */,
 				924F8C871EC8C71C00F7447A /* NSManagedObjectContext+Extension.swift in Sources */,
-				924F8C8B1EC8D27B00F7447A /* RankingTableViewCell.swift in Sources */,
```

Comment:
Do we need two table view cells here... or can we just have the single one? For a District Ranking we can just hide the WLT, and show it for an Event Ranking. I feel like they share enough code that we can make it one table view cell